packages/idempotency/src/Exceptions.ts

```diff
@@ -10,8 +10,23 @@ class IdempotencyInvalidStatusError extends Error {
 
 }
 
+class IdempotencyInconsistentStateError extends Error {
+
```

Comment:
Minor: adding a meaningful and easy to understand error message for each of these Errors would improve the DX and help troubleshooting errors

Reply:
I added a message to this when the error is created since this error can happen in more than one type of scenario. That way we can tailor the message by the scenario